application/templates/_helpers.tpl

```diff
@@ -10,11 +10,12 @@ Define the name of the chart/application.
 app: {{ template "application.name" . }}
 group: {{ .Values.labels.group }}
 provider: {{ .Values.labels.provider }}
+team: {{ .Values.labels.team }}
 {{- end -}}
 
 {{- define "application.labels.stakater" -}}
 {{ template "application.labels.selector" . }}
-version: "{{ .Values.labels.version }}"
+appVersion: "{{ .Values.labels.appVersion }}"
```

Comment:
Shouldn't appVersion be equal to Image tag, rather than having extra field?

Reply:
updated